pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -351,12 +351,44 @@ SELECT crdb_internal.validate_ttl_scheduled_jobs()
 
 query I
 SELECT count(1) FROM [SHOW SCHEDULES]
-WHERE label = 'row-level-ttl-$table_id'
+WHERE label = 'row-level-ttl-$table_name'
 ----
 0
 
 subtest end
 
+# Ensure that SHOW SCHEDULES's label column is updated to reflect the new table
```

Comment:
Could you add a test case where the schedule name uses id (mock with a manual update in test) and it's changed to use name after renaming the table? Just want to make sure your update query works.

Reply:
Done.